ibis/backends/duckdb/registry.py

```diff
@@ -28,6 +28,14 @@
 )
 from ibis.common.exceptions import UnsupportedOperationError
 
+try:
+    from ibis.backends.duckdb.datatypes import Geometry_WKB
+except ImportError:
+
+    class Geometry_WKB:
+        ...
+
+
```

Comment:
```suggestion
from ibis.backends.duckdb.datatypes import Geometry_WKB
```
we can rely on the import error handling in `datatypes.py` so only the import line is needed 

(it will also need to be  moved from where it is now that the try-except is gone or the linters will complain)